crates/re_viewer/data/quick_start_guides/cpp_connect.md

```diff
@@ -1,3 +1,45 @@
 ## C++ Quick Start
 
-TODO(ab): https://github.com/rerun-io/rerun/issues/3870
+${SAFARI_WARNING}
+
+#### Installing the Rerun viewer
+The Rerun C++ SDK works by connecting to an awaiting Rerun Viewer over TCP.
+
+Building from source:
+* [Install cargo](https://rustup.rs/)
+* `cargo install rerun-cli`
+
+Using `pip`:
+* `pip install rerun-sdk`
+
+After you've installed it, type `rerun` in your terminal to start the viewer.
+
+
+#### Using the Rerun C++ SDK
+First install the `arrow-cpp` library on your system using your favorite package manager.
+
+Then add the following to your `CMakeLists.txt`:
```

Comment:
```suggestion
If you're using CMake, you can add the following to your `CMakeLists.txt` to add a rerun_cpp target to your application. 
```
...

```
This download a bundle with pre-built Rerun C static libraries for most desktop platforms, all Rerun C++ sources and headers, as well as CMake build instructions for them.
```

Reply:
Thinking about it a bit more, maybe that's a separate section somewhere that documents how the Rerun C++ sdk is built up